morango/utils/uuids.py

```diff
@@ -50,6 +50,21 @@ def to_python(self, value):
             return value.hex
         return value
 
+    def get_default(self):
```

Comment:
Returned `None` for default because it would return empty string otherwise, and `calculate_uuid` method would not be called.